### DISCUSSION TITLE
EC2 Image, first time load issues

### DISCUSSION THREAD

**Discussion by tsabbay** [vercel/next.js#20087] [Help]
Hi,
**HELP= NEXT is blocking the event loop when an instance is first created.**

We are preparing for a major product release here and we noticed something that is disturbing us.
Our configuration is based on EC2 Images and we got auto-scaling based on this.

What we noticed that on the event of server start(not restart, the first-time start)   our node.js app is simply **freezing**.
It's not about hardware: 1% CPU is used, 10MB memory, very strong hardware.

We found that this is happening with **NEXT**, 1 minute that is blocking the node.js event loop.
We cant understand why and any help or ideas that can help us to find it will be appreciated.
Did anyone experienced the same behavior or **shed some light**?

**Comment by oonsamyi** [comment]
@tsabbay Hello!
I have the same issue. When I deploy the next.js server on production and open a page that has images, the server doesn't respond for about 1 minute. I think it is related to the generation of cache images: https://nextjs.org/docs/api-reference/next/image#caching-behavior

Have you found some solution?

**Comment by tsabbay** [comment]
No resolution yet...
